pii_recognition/registers/registry_test.py

```diff
@@ -0,0 +1,31 @@
+from typing import Any
+
+from .registry import Registry
+
+
+def test_Registry_add_item():
+    class ToyClass:
+        pass
+
+    actual = Registry[Any]()
```

Comment:
you mentioned that you only want class to be registered. I guess that's why you used `Any`? to represent the type of metaclass?

Reply:
Good point, I haven't thought about this. I just put `Any` there for annotation purpose letting people know it's a Generic class can take parameters. A better choice could be `Type`.

Regarding to the restrictions when registering, that comes in `add_item` where item must be `Type[T]`. Callable, int, str will make mypy complaining. Because it is a mypy error, I cannot really find a way to unittest it.